### ISSUE TITLE
backward compatibility isn't provided for subcommands of sonic_installer

### ISSUE THREAD

**Issue by stephenxs** [Azure/sonic-buildimage#5366]
**Description**

On master, all the underscore has been replaced by hash with backward. However, for subcommands of `sonic_installer`, backward compatibility isn't provided, which fails the deployment in the regression test.
```
root@r-boxer-sw01:/usr/lib/python2.7/dist-packages/config# sonic_installer binary_version
Usage: sonic_installer [OPTIONS] COMMAND [ARGS]...
Try "sonic_installer --help" for help.

Error: No such command "binary_version".
root@r-boxer-sw01:/usr/lib/python2.7/dist-packages/config# sonic-installer binary_version
Usage: sonic-installer [OPTIONS] COMMAND [ARGS]...
Try "sonic-installer --help" for help.

Error: No such command "binary_version".
```

**Steps to reproduce the issue:**
1. to run `sonic_installer binary_version`
2.
3.

**Describe the results you received:**
the command can't be executed

**Describe the results you expected:**
the command should be executed successfully.

**Additional information you deem important (e.g. issue happens only occasionally):**

    **Output of `show version`:**

    ```
    (paste your output here)
    ```

    **Attach debug file `sudo generate_dump`:**

    ```
    (paste your output here)
    ```

**Comment by jleveque** [comment]
@stephenxs: This was due to the fact that the aliases were not getting installed with the sonic-installer package. This bug should have been fixed by https://github.com/Azure/sonic-utilities/pull/1088. This PR was merged two weeks ago and the submodule has since been updated. Please try the an image built from the current master HEAD and confirm that the issue is resolved.

**Comment by stephenxs** [comment]
Thanks @jleveque . It is fixed by [Azure/sonic-utilities#1088](https://github.com/Azure/sonic-utilities/pull/1088).
